Expensify/App
# Add tap to view full screen image functionality

<Assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

<Explanation of the change or anything fishy that is going on>

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/142710 and https://github.com/Expensify/Expensify/issues/143523
### Tests
1. Upload an image and a PDF
2. Click on them and verify they open correctly

### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>


<img width="428" alt="Screen Shot 2020-10-30 at 1 51 19 PM" src="https://user-images.githubusercontent.com/30609178/97760848-688ccb80-1ac1-11eb-9d5d-3f68bf80a05f.png">
<img width="464" alt="Screen Shot 2020-10-30 at 1 51 07 PM" src="https://user-images.githubusercontent.com/30609178/97760856-6c205280-1ac1-11eb-81f6-c5939bd176f2.png">

![Screen Shot 2020-10-30 at 1 50 32 PM](https://user-images.githubusercontent.com/30609178/97760860-6fb3d980-1ac1-11eb-8581-64261fa50f86.png)

![Screen Shot 2020-10-30 at 1 50 40 PM](https://user-images.githubusercontent.com/30609178/97760866-73476080-1ac1-11eb-86ca-a826d8bd40b6.png)
